src/components/tools/ManipulatorTool.vue

```diff
@@ -22,8 +23,8 @@ export default defineComponent({
       required: true,
     },
     options: {
-      type: Object,
-      default: () => ({}),
+      type: Array as PropType<Array<any>>,
```

Comment:
We should probably strictly type this, but we can do that later. (We would also look to strictly typing the manipulatorClass.)

Reply:
ok, auto-created an issue regarding your comment so we can track this in the future.